crypto/bn/bntest.c

```diff
@@ -330,6 +335,15 @@ int main(int argc, char *argv[])
         goto err;
     (void)BIO_flush(out);
 #endif
+
+    /* silently flush any pre-existing error on the stack */
+    ERR_clear_error();
```

Comment:
@mattcaswell (cfr. https://github.com/openssl/openssl/pull/8294/files#r258680809)

Here i do `ERR_clear_error()` to prevent previously uncleared errors to confuse the user in case `test_ctx_consttime_flag()` fails.

I decided to do so given that if this point of main is reached none of the errors in the queue was being printed anyway before this commit.